orion/pkg/repos/migrations/000006_create_table_sessions.up.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE sessions (
+	id int unsigned NOT NULL AUTO_INCREMENT,
+	created_at datetime NOT NULL,
+	updated_at datetime NOT NULL,
+	deleted_at datetime,
+	class_id varchar(64) NOT NULL,
+	starts_at datetime NOT NULL,
+	ends_at datetime NOT NULL,
+	canceled boolean NOT NULL DEFAULT false,
```

Comment:
I believe in MySQL the default of a boolean is 0. false will work, but i think it is actually 0.